rest_framework/fields.py

```diff
@@ -1351,9 +1351,17 @@ class DurationField(Field):
         'overflow': _('The number of days must be between {min_days} and {max_days}.'),
     }
 
-    def __init__(self, **kwargs):
+    def __init__(self, *, format=empty, **kwargs):
         self.max_value = kwargs.pop('max_value', None)
         self.min_value = kwargs.pop('min_value', None)
+        if format is not empty:
+            if format is None or (isinstance(format, str) and format.lower() in (ISO_8601, DJANGO_DURATION_FORMAT)):
+                self.format = format
+            else:
+                raise ValueError(
+                    f"Unknown duration format provided, got '{format}'"
+                    " while expecting 'django', 'iso-8601' or `None`."
```

Comment:
Consider `'{ISO_8601}'`

Reply:
IMO it is just a waste of resources to put the `ISO_8601` variable here to be formatted since we already know its value and it is highly unlikely that the name of the standard is going to be changed.

Also it would be a point of having `{DJANGO_DURATION_FORMAT}` inside the format string, which from my perspective follows the same consideration.